README.md

```diff
@@ -88,6 +88,10 @@ There are two types of tests in this project:
 
 Both tests demonstrate how UTAM can be used to author and compile page objects, and how to integrate the UTAM runtime with WebdriverIO.
 
+## Dependency for Salesforce page objects
+
+The `package.json` file contains a dependency for the `salesforce-pageobjects` artifact, which contains page objects from Salesforce. The artifact is available on [npm](https://www.npmjs.com/package/salesforce-pageobjects).
```

Comment:
nit: I would replace artifact by package, even though technically correct in terms of the terminology devs are used to I think package is closer to what JS devs are used to speaking of the npm registry